app/code/Vladislav/ServiceContract/Model/VladislavSc.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vladislav\ServiceContract\Model;
+
+use Magento\Framework\Model\AbstractModel;
+use Vladislav\ServiceContract\Api\Data\VladislavScInterface;
+use Vladislav\ServiceContract\Model\ResourceModel\VladislavSc as VladislavScResource;
+use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\Model\AbstractModel;
```

Comment:
PHP syntax error: Cannot use Magento\Framework\Model\AbstractModel as AbstractModel because the name is already in use